ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -13,13 +13,27 @@ function (
         if (!initialised) {
             initialised = true;
 
-            window.injectCreative = injectCreative;
-            window.applyNativeFunctionCall('injectInlineCreative');
+            if (GU.opts.contentType === 'liveblog' || document.querySelector('.article__body--liveblog')) {
+              trackLiveBlogEpic();
+            } else {
+              window.injectCreative = injectCreative;
+              window.applyNativeFunctionCall('injectInlineCreative');
+            }
+        }
+    }
+
+    function trackLiveBlogEpic() {
+        var liveBlogEpicContainer = document.querySelector('contributions-epic');
+
+        if (liveBlogEpicContainer) {
+            var liveBlogEpicContainerId = liveBlogEpicContainer.getAttribute('id')
+
+            isCreativeInView(liveBlogEpicContainer, liveBlogEpicContainerId)
```

Comment:
trackLiveBlogEpic is called when a new liveblog blocks are added, however the creative may not be in view at this point, so by calling isCreativeInView(liveBlogEpicContainer, liveBlogEpicContainerId) here directly we'll not track if/when it does eventually scroll into view. We should replace the call to isCreativeInView directly and a listener as below...

window.addEventListener('scroll', util.debounce(isCreativeInView.bind(null, liveBlogEpicContainer, liveBlogEpicContainerId), 100));